Source/WebCore/platform/Scrollbar.cpp

```diff
@@ -28,8 +28,10 @@
 
 #include "DeprecatedGlobalSettings.h"
 #include "DisplayView.h"
+#include "Frame.h"
 #include "GraphicsContext.h"
 #include "LocalFrameView.h"
+#include "Page.h"
```

Comment:
This is a layering violation :(

Reply:
I'm going to move `deviceScaleFactor()` to `Widget` to resolve this.